src/main/java/Deadline.java

```diff
@@ -0,0 +1,13 @@
+public class Deadline extends Task {
```

Comment:
Consider documenting your classes to make it clearer what deadline, todo and event are meant to be.

Reply:
right this makes sense. thanks!